### ISSUE TITLE
Unclear when namechange is comitted to the server

### ISSUE THREAD

**Issue by Bios-Marcel** [scribble-rs/ScribblersPad#28]
While there's an "Ok" button as soon as you enter the textfield, it seems like leaving the menu is what triggers the namechange. I am wondering whether this could be weird for the player. I'd say it'd be okay to send it to the server right away, it's not that much traffic. This way for example, a player could change his name and directly ask their friends what they think about the name. Now they'd ask and their friends would be confused saying "your name didn't" change.

**Comment by BigETI** [comment]
So you are suggesting doing name changes while typing? That is absolutely possible.

**Comment by BigETI** [comment]
I mean it worked fine when I tested it, you could see in real-time when someone was changing their name.

**Comment by Bios-Marcel** [comment]
But it's unnecessary spamming, isn't it? I'd prefer to not do that.

**Comment by BigETI** [comment]
You are showing that in chat, don't you? Then yeah this might be spammy. :D

**Comment by Bios-Marcel** [comment]
Nah, I am not. I am talking about spamming network requests and causing constant update events on all clients.

But maybe displaying name changes in chat might not be a bad idea xD
